selfdrive/modeld/modeld.py

```diff
@@ -0,0 +1,307 @@
+#!/usr/bin/env python3
+import sys
+import time
+import logging
+import numpy as np
+from pathlib import Path
+from typing import Dict, Optional
+from setproctitle import setproctitle  # pylint: disable=no-name-in-module
+from cereal.messaging import PubMaster, SubMaster
+from cereal.visionipc import VisionIpcClient, VisionStreamType, VisionBuf
+from system.hardware import PC
+from common.params import Params
+from common.filter_simple import FirstOrderFilter
+from common.realtime import set_core_affinity, set_realtime_priority
+from selfdrive.modeld.models.commonmodel_pyx import ModelFrame, CLContext, Runtime # pylint: disable=import-error, no-name-in-module
```

Comment:
can probably get rid of these pylint suppressions, don't think ruff will error on this